src/components/global/tag.module.tsx

```diff
@@ -0,0 +1,23 @@
+import styles from "./tag.module.scss";
+
+// TODO semantics?, tag size (only 24px rn)
+
+function TagComponent({ children: input }: Props) {
```

Comment:
I'd simply name this as just `Tag`, same goes for `Button` and `Addon`. The usage of `<Tag>`, `<Button>` and `<Addon>`, etc. already indicates that it's a component. Kinda similar to naming a variable to hold a name `nameString`, instead of just `name`.

Reads a lot nicer too:
```tsx
<ButtonComponent variant="Secondary">official</ButtonComponent>
{/* versus */}
<Button variant="Secondary">official</Button>
```